packages/vanilla-extract/src/index.ts

```diff
@@ -9,34 +9,59 @@ import { precomputeValues } from '@capsizecss/core';
 import { ComputedValues, CreateStyleObjectParameters } from './types';
 import { capsizeStyle, capsizeVars } from './capsize.css';
 
-interface MediaQueries {
-  '@media': Record<string, CreateStyleObjectParameters>;
+interface ConditionalQueries {
+  '@media'?: Record<string, CreateStyleObjectParameters>;
+  '@container'?: Record<string, CreateStyleObjectParameters>;
 }
 
+const generateQueryVars = ({
+  queryType,
+  queries,
+}: {
+  queryType: '@media' | '@container';
```

Comment:
Thoughts on extracting this from the type?
```suggestion
  queryType: keyof ConditionalQueries;
```